src/core/Akka.Remote/Configuration/Remote.conf

```diff
@@ -523,6 +523,16 @@ akka {
       }
 
       ssl {
+        # When enabled, server will require client certificate during TLS handshake (mutual TLS)
+        require-client-certificate = off
+
+        # When enabled, client will present its certificate during TLS handshake (if available)
+        send-client-certificate = on
+
+        # When enabled, remoting will fail fast at startup if configured server certificate is unusable
+        # (e.g. missing private key or lacking Server Authentication EKU).
+        fail-fast-invalid-server-certificate = off
```

Comment:
Nah, this should never be configurable. Always fail if the cert is missing, the password doesn't match, or whatever. We already have the ability to suppress validation errors specified in the config - that should handle the legitimate edge cases like using self-signed certs. Delete this.